diesel_derives/src/lib.rs

```diff
@@ -863,6 +863,29 @@ pub fn derive_valid_grouping(input: TokenStream) -> TokenStream {
 /// # }
 /// ```
 ///
+/// # SQL Functions without Arguments
+///
+/// In diesel v1.4 sql functions without arguments used the `no_arg_sql_function!` macro,
+/// which has since been deprecated. The new `sql_function!` macro supports functions without
+/// arguments.
```

Comment:
I think the documentation should not mention old diesel versions, as this can (and probably will) stick around much longer than this is relevant for anyone anymore.

Reply:
Will do! Where do you recommend filling in the blanks for the change log docs around migrating? https://github.com/diesel-rs/diesel/blob/master/CHANGELOG.md#removed

Reply:
I think adding a section to [this part](https://github.com/diesel-rs/diesel/blob/1d17019315f868b3ec6bbdef2c1a98972e0f5de7/CHANGELOG.md#upgrade-notes) of the changelog would be great.